.travis.yml

```diff
@@ -24,11 +25,17 @@ jobs:
     - "3.6"
     install:
     - pip install -r dev-requirements.txt
+    - pip install -r requirements.txt
     - npm install @semantic-release/exec
     script:
     - git config --global user.email "releases@ladybug.tools"
     - git config --global user.name "ladybugbot"
     - npx semantic-release
+  - stage: docs
+    if: branch = master AND (NOT type IN (pull_request))
+    script:
+    - sphinx-apidoc -f -e -d 4 -o ./docs ./ladybug ./ladybug/euclid.py
```

Comment:
I should have probably asked this on the other PR too. How does documentation takes care of the new version of the documentation now?

Reply:
At the moment it will just overwrite all docs with the new version every time. This is why I mentioned using `readthedocs` on slack.

Reply:
but from where does it get the version? we used to pass the version in when running the command (https://github.com/ladybug-tools/honeybee-core/pull/17/files#diff-60254338249f657a0a83f98258a56bfeL13) but how does it know the version when it runs the command here?

Reply:
It doesn't. I can't think of a way to give it a version of we are going to make it rebuild the docs every time it runs deploy on master and also uses semantic release. That is unless we use readthedocs which builds docs based on git tags rather than for every commit.

Reply:
Got it! One more reason to try readthedocs then. Thank you for the quick reply.